tests/common/graph/test_nncf_graph.py

```diff
@@ -48,3 +49,46 @@ def test_find_matching_subgraphs():
             continue
         assert len(match) == 2
         assert match == nodes[:2]
+
+
+def test_parallel_edges():
+    nncf_graph = NNCFGraph()
+    nodes = []
+    for node in "abc":
+        nodes.append(nncf_graph.add_nncf_node(node, f"type_{node}", f"metatype_{node}"))
+
+    kwargs = {
```

Comment:
Same about `kwargs`

Reply:
Ok, I need to refactor graph.add_edge_between_nncf_nodes then

Reply:
Done